swoole_channel_coro.cc

```diff
@@ -36,7 +34,7 @@ typedef struct
     swLinkedList *consumer_list;
     bool closed;
     int capacity;
-    queue<zval> *data_queue;
+    std::queue<zval> *data_queue;
```

Comment:
In Solaris there's a `struct` named `queue` somewhere in its system headers, making `queue` ambiguous.
That's why we shouldn't `use namespace std;`